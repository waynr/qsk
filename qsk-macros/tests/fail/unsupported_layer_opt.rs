use qsk_macros::layer;

fn main() {
    layer!(
        ModLayer[Meow]: {
            Y -> HOME,
            F -> TT(Navigation),
        },
        Navigation: {
            END -> Exit(),
            Y -> HOME,
            U -> PAGEDOWN,
            I -> PAGEUP,
            O -> END,
            H -> LEFT,
            J -> DOWN,
            K -> UP,
            SEMICOLON -> RIGHT,
        },
    );
}
