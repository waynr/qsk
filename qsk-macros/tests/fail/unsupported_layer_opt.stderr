error: missing key code argument
 --> tests/fail/unsupported_layer_opt.rs:7:18
  |
7 |             F -> TT(Navigation),
  |                  ^^
